xmpp/xmpp.go

```diff
@@ -51,6 +51,10 @@ type item struct {
 	Topic           string `xml:"x>topic"`
 }
 
+type Ack struct {
+	Ack string `xml:"a"`
+}
+
```

Comment:
Ahh that was actually a legacy of the original approach I was going to take. I think it can actually go away.